packages/terra-navigation-layout/src/terra-dev-site/doc/example/NavigationLayoutDocCommon.scss

```diff
@@ -0,0 +1,33 @@
+:local {
+  .content-wrapper1 {
+    border: 4px dashed #d3d3d3;
+    height: calc(100% - 10px);
+    margin: 5px;
+    position: relative;
+    width: calc(100% - 10px);
+  }
+
+  .content-wrapper2 {
+    color: #808080;
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate3d(-50%, -50%, 0);
+  }
+
+  button {
```

Comment:
lets change this to a class, `.button`

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/742/commits/8262dbdbd969709d4d0a1cc42ba3188a1269b82c.